src/objects/api/kanalen.py

```diff
@@ -23,9 +23,11 @@ def __init__(
     def get_kenmerken(self, obj: models.Model, data: dict = None) -> dict:
         data = data or {}
         return {
-            kenmerk: data.get("type") or obj.object.object_type.url
```

Comment:
please in the future add black commits into `.git-blame-ignore-revs` as Sergei suggested